flake.nix

```diff
@@ -146,12 +146,12 @@
             enableDebug = false;
           });
           crypto3-clang = (pkgs.callPackage ./crypto3/crypto3.nix {
-            stdenv = pkgs.llvmPackages_18.stdenv;
+            stdenv = pkgs.llvmPackages_19.stdenv;
```

Comment:
Do we really need to update package?

Reply:
Just a general improvement.
I have selected llvm 18 before, just because it was the latest one, which could compile the current codebase and wrote me a note to upgrade to the latest one later